checker/lib/javadoc.astub

```diff
@@ -0,0 +1,107 @@
+import org.checkerframework.common.value.MinLen;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.qual.Pure;
+
+package com.sun.javadoc;
+
+// Index Checker annotations
+
+interface RootDoc {
```

Comment:
RootDoc appears twice in this stub file.  Does that work?
Is it necessary to separate the different types of annotations rather than making each class appear just once?

Reply:
The documentation in [sec. 28.4.1 of the manual](https://checkerframework.org/manual/#stub) indicates that when the same element occurs multiple times in a stub file (or in multiple different stub files), then the annotations are merged. So this does work.

A different question is whether we should separate the annotations. I chose to separate them because they were previously in annotated JDKs, where the source files for each checker are distinct. I separated the annotations in the stub file in a similar way, but one could argue that having all the annotations on a class/method in the same place is easier to read or use. I'm not partial to either position.

Reply:
OK, it's good that it works.  Thanks for checking that.

I don't think we should organize the file based on a previous organization, but what makes most sense in this file.  I favor having each class appear just once in the file.